api/src/main/java/edu/cornell/mannlib/vitro/webapp/web/templatemodels/individual/ExecutorFactory.java

```diff
@@ -0,0 +1,15 @@
+/* $This file is distributed under the terms of the license in LICENSE$ */
+
+package edu.cornell.mannlib.vitro.webapp.web.templatemodels.individual;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public final class ExecutorFactory {
+	
+	private static final ExecutorService executor = Executors.newCachedThreadPool();
```

Comment:
This will cause the application to not be unloaded and leak resources.

Threads created within the lifetime of the application need to be terminated within the lifetime of the application. If you are having a shared executor that lasts over the lifetime of the application, then you need to hook in a servlet context listener and ensure that the the executor is cleaned up when the context is destroyed.

Reply:
true